mscore/preferences.cpp

```diff
@@ -98,6 +98,7 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_EXPORT_MP3_BITRATE,                              new IntPreference(128, false)},
             {PREF_EXPORT_MUSICXML_EXPORTBREAKS,                    new EnumPreference(QVariant::fromValue(MusicxmlExportBreaks::ALL), false)},
             {PREF_EXPORT_MUSICXML_EXPORTLAYOUT,                    new BoolPreference(true, false)},
+            {PREF_EXPORT_MUSICXML_OUTDATED,                        new BoolPreference(false)},
```

Comment:
not sure whether `false` is the right default, it is the less advanced (Windows) users that have a problem with the renaming.

Reply:
That'd be possible, but would be quite confusing and likely to create traffic in the forum. So far there's only one setting where the default is platform dependant, `nativeDialogs`.